runtime/kusama/src/lib.rs

```diff
@@ -1461,8 +1484,13 @@ impl Get<Perbill> for NominationPoolsMigrationV4OldPallet {
 ///
 /// This contains the combined migrations of the last 10 releases. It allows to skip runtime
 /// upgrades in case governance decides to do so. THE ORDER IS IMPORTANT.
-pub type Migrations =
-	(migrations::V0940, migrations::V0941, migrations::V0942, migrations::Unreleased);
+pub type Migrations = (
+	migrations::V0940,
+	migrations::V0941,
+	migrations::V0942,
+	migrations::Unreleased,
+	init_state_migration::InitMigrate,
```

Comment:
Is there a reason why not to move that into `migrations::Unreleased,`?

`migrations::Unreleased` migrations do run, we just don't know yet in which of the upcoming Version it will be included.

At release time, merged `Unreleased` migrations will moved to a new `VXXXX` migration set.

Reply:
:+1: I will do, I just merged to master a bit too fast.
(currently I am trying to get better limit but encounter some limitation with try runtime, but will certainly get new limits soon).